### PR TITLE
S3 Upload Files CampaignApplication

### PR DESCRIPTION
**Pull request by Martbul**
Add functionality for uploading files to the cloud service when campaignApplication is created.

There is failing test in campaign.service.spec.ts in the master branch(releted to the creation of new contact list) - it is not related to my work or changes but it probably should be noted and fixed

### CODE REVIEW

File: apps/api/src/email/email.service.ts [hunk 1]
```diff
@@ -25,7 +25,6 @@ export class EmailService {
 
   async send(email: Email, settings: sg.MailDataRequired['mailSettings'] = {}): Promise<void> {
```
Comment (gparlakov) [inline apps/api/src/email/email.service.ts:26]: Let's not change files that are outside of the scope. Please revert this

File: apps/api/src/email/template.service.ts [hunk 2]
```diff
@@ -13,12 +13,14 @@ export class TemplateService {
     try {
       // pass it through mjml to produce html template
       const result = await this.getEmailTemplate(name)
+
```
Comment (gparlakov) [inline apps/api/src/email/template.service.ts:16]: ditto - redundant - please revert

File: apps/api/src/campaign-application/campaign-application.controller.ts [hunk 3]
```diff
@@ -27,18 +31,26 @@ export class CampaignApplicationController {
   ) {}
 
   @Post('create')
+  @UseInterceptors(
+    FilesInterceptor('file', 10, {
+      limits: { fileSize: 1024 * 1024 * 10 },
```
Comment (slavcho) [inline apps/api/src/campaign-application/campaign-application.controller.ts:36]: Please increase this to 30MB.
Sometimes the files can be quite big.

**Review by slavcho — APPROVED** [review]
